src/pages/home/components/SocialMedia.js

```diff
@@ -3,24 +3,35 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faFacebook, faTwitter, faInstagram } from '@fortawesome/free-brands-svg-icons';
 
 import config from 'config/constants';
-import styles from '../assets/socialmedia.scss';
 
 const SocialMedia = () => {
   return (
-    <div className={styles.socialMedia}>
-      <div className={styles.iconWrapper}>
+    <div className="flex flex-row-reverse self-center md:self-auto">
+      <div className="group m-3 p-4 rounded-full border border-solid border-primary-light cursor-pointer hover:bg-primary-light transition transition-fast transition-property-all transition-timing-linear">
         <a target="_blank" href={config.social.FB_URL} rel="noreferrer noopener">
-          <FontAwesomeIcon icon={faFacebook} size="2x" className={styles.greenIcons} />
+          <FontAwesomeIcon
+            icon={faFacebook}
+            size="2x"
+            className="text-primary-light group-hover:text-white transition transition-fast transition-property-all transition-timing-linear"
+          />
         </a>
       </div>
-      <div className={styles.iconWrapper}>
+      <div className="group m-3 p-4 rounded-full border border-solid border-primary-light cursor-pointer hover:bg-primary-light transition transition-fast transition-property-all transition-timing-linear">
         <a target="_blank" href={config.social.INSTA_URL} rel="noreferrer noopener">
-          <FontAwesomeIcon icon={faInstagram} size="2x" className={styles.greenIcons} />
+          <FontAwesomeIcon
+            icon={faInstagram}
+            size="2x"
+            className="text-primary-light group-hover:text-white transition transition-fast transition-property-all transition-timing-linear"
+          />
         </a>
       </div>
-      <div className={styles.iconWrapper}>
+      <div className="group m-3 p-4 rounded-full border border-solid border-primary-light cursor-pointer hover:bg-primary-light transition transition-fast transition-property-all transition-timing-linear">
```

Comment:
<img width="493" alt="screen_shot_2018-11-19_at_6_30_31_am" src="https://user-images.githubusercontent.com/219207/48707560-7fcc9a00-ebc5-11e8-8bfd-a2727e017b08.png">

Finally please remove the margin-right from the last item.